src/util/source_location.cpp

```diff
@@ -93,3 +94,25 @@ std::ostream &operator << (
   out << source_location.as_string();
   return out;
 }
+
+/*******************************************************************\
+
+Function: is_build_in
```

Comment:
Doesn't seems like it , I did a quick fuzzy search through the code-base and found a list of classes (listed below) that are basically partiality replicating the functionality above every time it is needed.

src/goto-programs/graphml_witness.cpp
src/goto-instrument/count_eloc.cpp